node.go

```diff
@@ -251,28 +251,84 @@ func (n *Node) RequestNodeInformationFrame() error {
 	return err
 }
 
+// To optimize the interview process we'll be interview command classes
+// in the following order:
+// 1) SecurityCommandClass
+// 2) VersionCommandClass
+// 3) Remaining insecure command classes
+// 4) Secure command classes
+func commandClassesInOrderToInterview(set cc.CommandClassSet) []cc.CommandClassID {
+	insecure := set.ListBySecureStatus(false)
+
+	securityccidx := -1
+	versionccidx := -1
+	remaininginsecure := []cc.CommandClassID{}
```

Comment:
Wouldn't this contain both secure and insecure command classes?

Reply:
No because we're iterating over the slice `insecure` which is initialized with the result of `ListBySecureStatus(false)`. The `ListBySecureStatus` function filters the support command classes for a given node and only returns the ones with the security status you specify i.e. true/false.

Reply:
Ah, I understand now. Perfect.